central/policy/customresource/custom_resource_test.go

```diff
@@ -143,3 +144,14 @@ func TestToDNSSubdomainName(t *testing.T) {
 		})
 	}
 }
+
+// generateCustomResource generate custom resource in YAML text from a policy
+func generateCustomResource(policy *storage.Policy) (string, error) {
+	w := &bytes.Buffer{}
+	convertedPolicy := ConvertPolicyToCustomResource(policy)
```

Comment:
nit: policyCR for var name for readability

Reply:
I will remove it. :)